### PR TITLE
Remove sensitive alerts from datasets

### PR DESCRIPTION
**Pull request by MaksimZhukov**
In scope of this PR we have removed the following SpringBoard alerts from database since it may affect user's sensitive data:

1. Sign In to App Store
2. Sign In to Game Center
3. Sign In to iTunes Store
4. Sign In to iMessage
5. Sign In to FaceTime
6. Sign In to iCloud
7. Sign In with Apple ID
8. The current Wi-Fi network and others nearby will be disconnected until tomorrow.\n\nWi-Fi will continue to be available for AirDrop, Personal Hotspot, and location accuracy.
9. The current WLAN network and others nearby will be disconnected until tomorrow.\n\nWLAN will continue to be available for AirDrop, Personal Hotspot, and location accuracy.
10. Currently connected accessories will be disconnected and other accessories will not connect.\n\nBluetooth will continue to be available for Apple Watch, Apple Pencil, Personal Hotspot, and Handoff.
11. Turn On Bluetooth to Allow “%@” to Connect to Accessories

Also we have fixed tests in `SpringBoardAlertsCurrentLanguageTests.m`

### CODE REVIEW

**Review by maxim-lobanov — APPROVED** [review]

File: Server/Resources.xcassets/springboard-alerts/springboard-alerts-sk.dataset/alerts.json [hunk 1]
```diff
@@ -241,41 +241,13 @@
...
-    ],
-    "shouldAccept": true
-  },
   {
     "title": "Prihláste sa do iTunes Storu",
     "buttons": [
       "Zrušiť"
     ],
     "shouldAccept": true
   },
```
Comment (ivan-nosar) [inline Server/Resources.xcassets/springboard-alerts/springboard-alerts-sk.dataset/alerts.json]: Shouldn't it be deleted too?

File: Server/Resources.xcassets/springboard-alerts/springboard-alerts-fr.dataset/alerts.json [hunk 2]
```diff
@@ -234,48 +234,6 @@
     ],
     "shouldAccept": true
   },
```
Comment (ivan-nosar) [inline Server/Resources.xcassets/springboard-alerts/springboard-alerts-fr.dataset/alerts.json]: `Sign In with Apple ID` missed here. Please double check

File: Server/Resources.xcassets/springboard-alerts/springboard-alerts-zh_CN.dataset/alerts.json [hunk 3]
```diff
@@ -290,27 +248,13 @@
...
-    ],
-    "shouldAccept": true
-  },
   {
     "title": "当前无线局域网络及附近其他网络会在明天之前保持断开状态。\n\n无线局域网仍可用于“隔空投送”、“个人热点”和定位准确性。",
     "buttons": [
       "好"
     ],
     "shouldAccept": true
   },
```
Comment (ivan-nosar) [inline Server/Resources.xcassets/springboard-alerts/springboard-alerts-zh_CN.dataset/alerts.json]: I believe it should be deleted too

File: tools/springboard-alerts/frameworks.json [hunk 4]
```diff
@@ -91,23 +91,6 @@
             { "title": "“%@” Wants to Use “%@” to Sign In", "button": "Continue" }
         ]
     },
-    {
```
Comment (ivan-nosar) [inline tools/springboard-alerts/frameworks.json]: I count 10 deletions here, but we have to delete 11 alerts. Am I missed something?

File: Server/Resources.xcassets/springboard-alerts/springboard-alerts-sk.dataset/alerts.json (hunk 1 above)
Comment (MaksimZhukov) [reply Server/Resources.xcassets/springboard-alerts/springboard-alerts-sk.dataset/alerts.json]: Yes, thanks!

File: Server/Resources.xcassets/springboard-alerts/springboard-alerts-zh_CN.dataset/alerts.json (hunk 3 above)
Comment (MaksimZhukov) [reply Server/Resources.xcassets/springboard-alerts/springboard-alerts-zh_CN.dataset/alerts.json]: Yes, thanks!

File: Server/Resources.xcassets/springboard-alerts/springboard-alerts-fr.dataset/alerts.json (hunk 2 above)
Comment (MaksimZhukov) [reply Server/Resources.xcassets/springboard-alerts/springboard-alerts-fr.dataset/alerts.json]: This file does not include this alert. Thanks for the point

**Review by ivan-nosar — APPROVED** [review]

**Review by maxim-lobanov — APPROVED** [review]

**Review by jmoody — APPROVED** [review]

**Review by vsafonkin — APPROVED** [review]

**Review by AndreyMaslennikov — APPROVED** [review]

**Pull request merged**